src/data/projects/newrelic-ruby-agent.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "newrelic-ruby-agent",
+  "fullName": "newrelic/newrelic-ruby-agent",
+  "slug": "newrelic-ruby-agent",
+  "owner": {
+    "login": "newrelic",
+    "type": "Organization"
+  },
+  "title": "Ruby Agent",
+  "supportUrl": "https://discuss.newrelic.com/c/support-products-agents/ruby-agent",
+  "githubUrl": "https://github.com/newrelic/newrelic-ruby-agent",
+  "permalink": "https://opensource.newrelic.com/projects/newrelic/newrelic-ruby-agent",
+  "iconUrl": null,
+  "shortDescription": "New Relic's Ruby Agent",
+  "description": "The New Relic Ruby agent monitors your applications to help you identify and solve performance issues. You can also extend the agent's performance monitoring to collect and analyze business data to help you improve the customer experience and make data-driven business decisions.",
```

Comment:
This is just an aesthetic thing, but a shorter description might look better (this one gets cut off on the project page).

![image](https://user-images.githubusercontent.com/6722433/87357331-c48f0c80-c531-11ea-95c1-07d2d79d8111.png)

![image](https://user-images.githubusercontent.com/6722433/87357353-d40e5580-c531-11ea-9b31-0396f07659b0.png)